examples/libvirtgo_events/main.go

```diff
@@ -0,0 +1,55 @@
+// Copyright 2016 The go-qemu Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"flag"
+	"fmt"
+	"log"
+
+	"github.com/digitalocean/go-qemu/qmp"
+)
+
+var (
+	uri        = flag.String("uri", "qemu:///system", `URI to connect to the libvirtd host.`)
```

Comment:
Nitpick, use quotes instead of backticks since we don't need quotes inside the string.